cirq/value/duration_test.py

```diff
@@ -84,10 +84,6 @@ def test_cmp():
     assert (Duration() != 0)
 
 
-# In python 2, comparisons fallback to __cmp__ and don't fail.
-# But a custom __cmp__ that does fail would result in == failing.
-# So we throw up our hands and let it be.
```

Comment:
This raises the question: do we want to implement comparability of `Duration` (and `Timestamp`) with other types now that we're off python 2? If so, we should file an issue to track before removing this comment.

Reply:
CC @Strilanc

Reply:
What other values could timestamp be compared with?

I don't see any reason not to define comparison between duration and the builtin timedelta type.